data/hazards.txt

```diff
@@ -99,3 +70,65 @@ hazard "Small Black Hole Gravity"
 		"damage dropoff" 120 1700
 		"dropoff modifier" 0
 
+
+hazard "Core Exposed Heat"
+	"constant strength"
+	"range" 50000
+	weapon
+		"burn damage" 0.06
+		"relative heat damage" 0.0012
+#replace 0.0006 rel. heat damage with 0.00006 rel. burn damage if/when implemented.
+
+hazard "Core Heat Flare"
+	"duration" 10 60
+	"strength" 0.5 3
+	"range" 50000
+	weapon
+		"heat damage" 50
+		"burn damage" 1
+		"relative heat damage" 0.04
```

Comment:
It's actually not possible to build a ship capable of handling this - this hazard imparts an average of 2.45 relative heat and potentially as high as 7.2. 
We don't track negative heat, so more than 2 relative heat (assuming it's halved while shields are up) will _always_ overheat you, regardless of your ship, unless you have something like an archon with absurdly high heat dissipation. 
I did some testing, and even with a ship cooled with passive cooling such that its stated idle temperature is -15k and a heat dissipation of 2 (almost every ship in the game has less than 1) I still ended up overheating a lot while travelling around the core, and was still permanently overheated in Sag A*.

Reply:
You should be overheating a lot I had no protection and could manage jumping out.

Reply:
3000 heat damage per second, plus 600 burn damage per second (which will impart a total of 600 heat over a long period, and an average of 60 per second to begin with, and if it builds up, up to 6000 per second I think) plus 240% of your maximum heat capacity per second.

You'll need a lot of cooling to spend any appreciable time here, and you're probably not supposed to ever be able to simply idle in here. If you wanted to, though, you'd need about -9k idle heat to start, plus -240 idle heat per ton of mass on your ship. That'd do it. (That resolves to -249,000 idle heat on a ship of 1000t. -15,000 is peanuts compared to that!)

Reply:
With large heat shunts, reaching -249,000 idle heat would require 77 large heat shunts, or 1,848 outfit space - which puts your mass up to nearly twice that 1000t figure, even before considering the mass of any other outfits and the ship itself. So I think it's genuinely impossible (with the cooling that exists in-game currently, at least) to build a ship that can resist that.

Reply:
These heat and burn are totally not permanent they only last for a few seconds

Reply:
You should not expect to spend any significant time in the core without active cooling.
But there are two tiers of system hazards and the goal is that *with specialised outfitting* you can chill in the less aggressive systems somewhat manageably.

Reply:
Active cooling doesn't behave differently from cooling in any way that would make it more useful here, and all the active cooling outfits in the game currently are less space-efficient than the large heat shunt when you account for the energy requirement, so active cooling isn't going to change anything.
I don't think it's going to be possible to get the Xapleaux ships to be able to handle the heat without either giving them massively better heat dissipation than other ships or implementing outfits that completely break the balance of the game.

Based on the numbers mentioned here previously, currently a ship needs - with the best cooling in the game currently - to devote 184.8% of its _mass_ (not outfit space) to operate here - you'd need to get that below 50% (unless Xapleaux ships have an atypically high outfit space : hull mass ratio, which they seem not to based on the current stats) for it to be _possible_ to build Xapleaux ships to handle it, and probably more like 25% if they're going to be in any way functional.

(Also, I think the extra heat capacity on the piezoelectric shaft may actually harm the Xapleaux here, since it'll increase the amount of heating from that relative heat damage)

Reply:
If that is indeed the case then yes, changes will need to be made. I will defer to Fero as the heat expert for the moment though. I won't get to dive into this stuff for a little while.

Reply:
So, I don't expect any ship to simply "exist" around in the core. My point was "woah, -250,000 heat is an absurd number, so maybe I should stay away from there"

There are some places that ships can't be expected to withstand. The bottom of an ocean, the deep atmosphere of a gas giant or a star, the stellar neighborhood of the galactic core.

When the numbers come out to "a ship would have to dedicate nearly 2x its mass solely to cooling to be able to exist within the core", many things should come under question, though I think what should come under question is "should we go fly in there?" rather than "did the devs make a mistake typing in these numbers?"

Reply:
(Important to note, this is the unshielded, fully inhospitable hazard. The "core sheltered" is used in systems which have nebular clouds around them, shielding them from their neighborhood, severely reducing their danger. Though, this is the core, the danger is always present to an extent)

Reply:
Their ships arr able to manage this heat actually, for the most part, because of their good manoeuvrability
Only saw them getting stuck once and like me they got unstucked

Reply:
> Their ships arr able to manage this heat actually, for the most part, because of their good manoeuvrability Only saw them getting stuck once and like me they got unstucked

They may struggle a lot more once these systems have arrival distances set, since then they'll often have to cross most of the system in order to land.

Reply:
I'm pretty sure that these dont happen in systems where there are inhabited planets